wesnoth-thot.po

```diff
@@ -1715,8 +1696,11 @@ msgid ""
 "state. Once that is done, none will be able to stand before me! Join us, "
 "loremaster. We can put your talent to good use."
 msgstr ""
+"Kladivo je stále poškozené. Potřebujeme znalosti obřadníka, abychom zbývající "
+"části znovu vykovali a artefakt tak do nejsilnějšího stavu navrátili. Až "
```

Comment:
```suggestion
"části znovu vykovali a artefakt tak navrátili do nejsilnějšího stavu. Až "
```

Reply:
Ale aspoň jednou by to tam zůstat mohlo.

Reply:
zůstane (vykovali)